src/main/java/edu/kis/powp/jobs2d/features/DrawOnFreePanelFeature.java

```diff
@@ -5,11 +5,9 @@
 
 public class DrawOnFreePanelFeature {
 
-	private static Application app;
```

Comment:
You have forgotten to remove Application from imports (line 3)

Reply:
It's is used in setupButtonClick() function as parameter